scale/scale.go

```diff
@@ -0,0 +1,92 @@
+package scale
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/restclient"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+var podNamespace = os.Getenv("POD_NAMESPACE")
```

Comment:
This is the only runtime parameter that is configurable via an env var.  I suggest moving it to a CLI flag and doing interpolation via the Pod spec.  ie:
```
/kube-sqs-autoscaler --kubernetes-namespace=$(POD_NAMESPACE)
```

That way all options are in the same place.  Just more of an operational thing.